### PR TITLE
Introduce new SIG governance and tooling

### PR DESCRIPTION
**Pull request by lyarwood**
This PR is the outcome of the `SIGs - What's Next?` calls happening below:

https://docs.google.com/document/d/1yOtxWS2eIgA2VT71NbFaDEzGyVfmiDCwoRBHKow0K6c/edit

At present this series aims to:

- Introduce a [Governance SIG overview](https://github.com/kubevirt/community/pull/236/commits/98ab1f40b05c512cc171965745e6965126bb49f0)
- [Seeds SIG documentation from k8s](https://github.com/kubevirt/community/pull/236/commits/95fa7c26dc711d445d8ef80cfb20481e55b0582f) 
-  [updates this documentation for KubeVirt](https://github.com/kubevirt/community/pull/236/commits/663b4cf9f767ad982882204b8a7a77ab9ea13d64)
- [Seeds the project with SIG management tooling from k8s](https://github.com/kubevirt/community/pull/236/commits/6fe53ede7ee07ab76aa6877b6968ec0d312a0a3d)
- [Generates some example directories](https://github.com/kubevirt/community/pull/236/commits/b410205f4971b2b02a930475d7ef91dd51e883cf)

### CODE REVIEW

**Comment by dhiller** [comment]
@lyarwood please note that the [sigs.yaml file is required to update](https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/community/community-presubmits.yaml#L6) the [CNCF stats for KubeVirt](https://kubevirt.devstats.cncf.io/d/8/dashboards).

I will need to check whether the changes impact the stats generation.

/assign

File: sigs.yaml
Comment (dhiller) [inline sigs.yaml:36]: if you intend to point out that there's a storage association inside kubevirt (which IMHO is true), then please add pointers to owners files concerning storage portions inside kubevirt/kubevirt in the below owners section.

Example (note that this OWNERS file doesn't exist yet):
* https://github.com/kubevirt/kubevirt/blob/61853c7e26d6a38a666ffa4bdd6f6cf3fe8b4afe/pkg/storage/OWNERS

**Review by dhiller — CHANGES_REQUESTED** [review]

File: sigs.yaml
Comment (dhiller) [reply sigs.yaml:7]: Background on this: we are generating associations for sig activities inside [kubevirt devstats](https://kubevirt.devstats.cncf.io/) with these entries.

See [kubevirt/community](https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/community/community-postsubmits.yaml#L3) postsubmit for more details

Comment (lyarwood) [reply sigs.yaml:7]: ACK understood but I think we need to look at moving lots of these deprecated/stale projects elsewhere.

File: sigs.yaml
Comment (lyarwood) [reply sigs.yaml:36]: ACK understood

File: sigs.yaml
Comment (dhiller) [reply sigs.yaml:7]: Or maybe we should contact project maintainers and ask them whether it would be safe to archive them? 

Note: Archiving can be done over kubevirt/project-infra orgs.yaml

File: sigs.yaml
Comment (dhiller) [reply sigs.yaml:36]: BTW, why do you want to remove the cdi subproject here?

Comment (lyarwood) [reply sigs.yaml:36]: Reverted, I thought we were using `kubevirt` as a generic subproject when listing `OWNERS` files from multiple different projects.

File: sigs.yaml
Comment (lyarwood) [reply sigs.yaml:7]: Reverted, yeah we really should but I'm not going to tackle that as part of this effort.

Comment (dhiller) [reply sigs.yaml:7]: Created https://github.com/kubevirt/community/issues/239 as a follow up.

Comment (lyarwood) [reply sigs.yaml:7]: Okay I lied, I'll look at this for some of the obvious projects under https://github.com/kubevirt/project-infra/pull/2942

File: GOVERNANCE.md
Comment (iholder101) [inline GOVERNANCE.md:1]: WDYT about adding a section mentioning that for a certain code change the contributor would possibly need an approval from multiple SIGs' representatives? For example, a PR might change both network and storage related code, so they would need an approval from both a network and a storage approver. This is how it works in k8s

File: sigs.yaml
Comment (dhiller) [inline sigs.yaml:4]: label declaration missing

**Review by dhiller — APPROVED** [review]
I've noticed that we are missing some labels in the config, therefore I've created a WIP PR: https://github.com/kubevirt/project-infra/pull/2953

Other than that looks good to me.

File: GOVERNANCE.md
Comment (aburdenthehand) [reply GOVERNANCE.md:57]: Makes sense. 
's/Every identifiable subpart/Most identifiable subparts/' and then "intended" does the rest.

File: GOVERNANCE.md
Comment (aburdenthehand) [inline GOVERNANCE.md:69]:
```suggestion
A primary reason that SIGs exist is as forums for collaboration. Most of the work in a SIG should stay local within that SIG. However, SIGs must communicate in the open; ensure other SIGs and community members can find notes of meetings, discussions, designs, and decisions; and periodically communicate a high-level summary of the SIG's work to the community.
```

File: sigs.yaml
Comment (lyarwood) [reply sigs.yaml:7]: As above, this is dropped for now.

File: sigs.yaml
Comment (lyarwood) [reply sigs.yaml:68]: ACK thanks.

File: sigs.yaml
Comment (lyarwood) [reply sigs.yaml:50]: ACK thanks.

File: GOVERNANCE.md
Comment (lyarwood) [reply GOVERNANCE.md:57]: ACK thanks I've used @aburdenthehand 's suggestion.

File: GOVERNANCE.md
Comment (lyarwood) [reply GOVERNANCE.md:1]: I've added the following:

**Comment by lyarwood** [comment]
/hold

I'm likely going to break this up into a few different PRs after the call tomorrow.

**Comment by lyarwood** [comment]
/unhold

As discussed in the latest `SIGs - What's Next?` [1] meeting I think the PR is getting to the stage now where we should land what we have and work to refine it through engaging the existing SIGs to populate the required metadata.

I'm hopeful that this process will help us iron out any issues and nits with the process instead of this PR forever being in review and not moving the overall SIG process forward.

Once merged I'm going to use the `SIGs - What's Next?` call to help on-board folks into the process and set out what is required going forward.

[1] https://docs.google.com/document/d/1yOtxWS2eIgA2VT71NbFaDEzGyVfmiDCwoRBHKow0K6c/edit

File: tools/sigs/generator/README.md
Comment (aburdenthehand) [inline tools/sigs/generator/README.md:7]:
```suggestion
This folder contains scripts to automatically generate documentation about the
different Special Interest Groups (SIGs) of KubeVirt. The authoritative
```
?

File: tools/sigs/generator/README.md
Comment (aburdenthehand) [inline tools/sigs/generator/README.md:21]: Using UTC removes the issue of daylight savings, and the KubeVirt calendar updates times depending on the user/browser location.

File: tools/sigs/generator/annual-report/github_issue.tmpl
Comment (aburdenthehand) [inline tools/sigs/generator/annual-report/github_issue.tmpl:1]: Do we need these three annual report files?

File: tools/sigs/generator/committee_readme.tmpl
Comment (aburdenthehand) [inline tools/sigs/generator/committee_readme.tmpl:67]: I'd say we can drop these lines

File: tools/sigs/generator/liaisons.tmpl
Comment (aburdenthehand) [inline tools/sigs/generator/liaisons.tmpl:1]: As before, I don't think we need specific liaisons

File: tools/sigs/generator/sig_readme.tmpl
Comment (aburdenthehand) [inline tools/sigs/generator/sig_readme.tmpl:56]:
```suggestion
- Slack: [#{{.Contact.Slack}}](https://kubernetes.slack.com/messages/kubevirt-dev{{.Contact.Slack}})
```

**Review by aburdenthehand — COMMENTED** [review]
Great work, @lyarwood 
I've done a run through and picked up some changes re: steering committee and some kubernetes info, and updated the slack channels etc. I've also tried to slim down some of the weight that I don't think applies to our project.

**Comment by lyarwood** [comment]
/close

See https://groups.google.com/g/kubevirt-dev/c/dXvwU6Pon7Y for more context.

**Pull request closed without merging**
